src/rpc/util.cpp

```diff
@@ -2,6 +2,7 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#include <clientversion.h>
```

Comment:
Not sure. Wouldn't it be better to use one of the existing checks or formatters from `util/check`?

Reply:
Maybe? Could just use `StrFormatInternalBug` I guess, but the file/line/func info would be kind of misleading as to where the bug is?

(As it is, it backports cleanly to 24.0, which was convenient for what I was trying to debug)